scripts/modules/ui.js

```diff
@@ -48,14 +49,22 @@ function displayBanner() {
 	// Read version directly from package.json
 	let version = 'unknown'; // Initialize with a default
 	try {
-		const packageJsonPath = path.join(process.cwd(), 'package.json');
+		// Get the directory of the current module (ui.js)
+		const currentModuleFilename = fileURLToPath(import.meta.url);
+		const currentModuleDirname = dirname(currentModuleFilename);
+		// Construct the path to package.json relative to ui.js (../../package.json)
+		const packageJsonPath = pathJoin(currentModuleDirname, '..', '..', 'package.json');
+
 		if (fs.existsSync(packageJsonPath)) {
-			const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+			const packageJsonContent = fs.readFileSync(packageJsonPath, 'utf8');
+			const packageJson = JSON.parse(packageJsonContent);
 			version = packageJson.version;
+		} else {
+			log('warn', `Own package.json not found at expected path: ${packageJsonPath}`);
```

Comment:
feels like you're repeating yourself, would be nice to move this whole finding package.json and checking version into a separate util that we can import here and in commands.js and re-use